modules/manage/activate-admin.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+$activate_url = wp_nonce_url(
+		Jetpack::admin_url(
+			array(
+				'page'   => 'jetpack',
+				'action' => 'activate',
+				'module' => 'manage',
+			)
+		),
+		"jetpack_activate-manage"
+	);
+
+$section = isset( $_GET['section'] ) ? $_GET['section'] : null;
+
+switch( $section ) {
+	case 'plugins':
```

Comment:
The wording seems a bit weird for these notices. It might be worth reviewing them all, as it seems like we're begging folks to enable the module right now.